feature/google/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -572,15 +624,15 @@ class BillingWrapperTest {
     }
 
     @Test
-    fun `purchase fails if purchase option does not have token`() {
+    fun `subscription purchase fails if purchase option is not GooglePurchaseOption`() {
```

Comment:
do we have the same test for a non-googlestoreproduct?

Reply:
Yeah, that was added before: https://github.com/RevenueCat/purchases-android/blob/bc5-support/feature/google/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt#L621

Reply:
cool thank you!